slam-libraries/viam-cartographer/README.md

```diff
@@ -49,7 +49,6 @@ brew install openssl eigen gflags glog suite-sparse sphinx-doc pcl
 
 ```
 
-
 #### Setup: Raspberry Pi OS Lite (64-bit)
 
 **Install dependencies**
```

Comment:
Could follow the orbslam [README](https://github.com/viamrobotics/slam/blob/main/slam-libraries/viam-orb-slam3/README.md) and include the automatic installation instructions:

./setup_cartographer.sh

if desired. 

Note: all make commands are covered in the slam/slam_libraries/README.md file.